.github/workflows/build_ci.yml

```
@@ -186,44 +186,49 @@ jobs:
     timeout-minutes: 8
     steps:
     - name: Checkout
+      if: ${{ needs.get-next-version.outputs.new-release-version }}
```

I think it's more clear to use this variable : 

```suggestion
      if: ${{ needs.get-next-version.outputs.new-release-published }}
```